ddl/db_integration_test.go

```diff
@@ -2152,20 +2150,24 @@ func (s *testIntegrationSuite7) TestCreateExpressionIndexError(c *C) {
 }
 
 func (s *testIntegrationSuite7) TestAddExpressionIndexOnPartition(c *C) {
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.Experimental.AllowsExpressionIndex = true
+	})
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t;")
-	s.tk.MustExec(`create table t(
+	tk.MustExec(`create table t(
 	a int,
 	b varchar(100),
-	c int,
+	c int)
 	PARTITION BY RANGE ( a ) (
 	PARTITION p0 VALUES LESS THAN (6),
 		PARTITION p1 VALUES LESS THAN (11),
 		PARTITION p2 VALUES LESS THAN (16),
 		PARTITION p3 VALUES LESS THAN (21)
 	);`)
 	tk.MustExec("insert into t values (1, 'test', 2), (12, 'test', 3), (15, 'test', 10), (20, 'test', 20);")
+	// TODO: Uncomment lines below after fixing #18132.
```

Comment:
Please remove this comment.

Reply:
Fixed in https://github.com/pingcap/tidb/pull/18003/commits/476e4d00c672f3b82c4c1e406ee519d0d2cdb4b4, PTAL